travis/samples/.jhipster/EntityWithPaginationDTO.json

```diff
@@ -8,7 +8,7 @@
         }
     ],
     "changelogDate": "20160806123743",
-    "entityTableName": "entity_with_pagination_and_dto",
+    "entityTableName": "entity_with_pagination_dto",
```

Comment:
What if users create entities with long name?

Reply:
Doesnt prettier has a setting to break imports into multiline when line limit exceeds? Else users will get error when they create something long

Reply:
In fact Prettier fails at this -> it puts all imports in one line

Reply:
Then if your import is more than 180 chars, you named it wrong

Reply:
ya but since the imports will have path as well its quite easy to reach the limit even with something like `CustomerCategoryServiceCode`, and if there are are more than one item imported then it easily will reach 180

Reply:
I thought prettier was smart enough to handle these, may be we should use `// prettier-ignore` comment for these as we already handle breaking stuff when its over 180 chars

Reply:
We can also put the file that <code>import</code> entities in <code>.prettierignore</code>  if it is the only one that break tslint rules.
Or there will be a new <a href='https://github.com/palantir/tslint/pull/3099'>tslint feature<a> in the next release that we can use to ignore <code> import</code>  lines

Reply:
most files will be importing entities, hence I dont how ignoring entire file will work